src/applications/vaos/services/appointment/transformers.js

```diff
@@ -125,17 +128,22 @@ export function transformVAOSAppointment(appt, useFeSourceOfTruthTelehealth) {
 
   if (isRequest) {
     const { requestedPeriods, created } = appt;
-    const reqPeriods = requestedPeriods?.map(d => ({
-      // by passing the format into the moment constructor, we are
-      // preventing the local time zone conversion from occuring
-      // which was causing incorrect dates to be displayed
-      start: `${moment(d.start, 'YYYY-MM-DDTHH:mm:ss').format(
-        'YYYY-MM-DDTHH:mm:ss',
-      )}.000`,
-      end: `${moment(d.end, 'YYYY-MM-DDTHH:mm:ss').format(
-        'YYYY-MM-DDTHH:mm:ss',
-      )}.999`,
-    }));
+    const reqPeriods = requestedPeriods?.map(d => {
+      const endDate = d.end || d.start;
+
+      return {
+        start: `${formatInTimeZone(
```

Comment:
nit: not necessary for this PR but we might want to consider keeping this as a Date object instead of a string. Might be quite a bit to refactor though so we can do this separately at a later time.